File: optidivy/src/model/Lense.java
package model;

public class Lense {
    int id;
    String type;
    double price;

    public Lense(int id, String type, double price) {
        this.id = id;
        this.type = type;
        this.price = price;
    }

    @Override
    public String toString() {
        return type + ": " + price ;
    }
}

File: app/optidivy/app/Models/Inventory/Lense.php
<?php

namespace App\Models\Inventory;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Lense extends Model
{
    public function stock(): BelongsTo
    {
        return $this->belongsTo(\App\Models\Inventory\Stock::class);
    }
}
